prisma/migrations/20220121222436_add_versions/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- CreateTable
```

Comment:
Can the migration be `create_versions` instead of `add_versions` for consistency?